Costing doc for stoichometric reactor (#1449)
watertap-org/watertap: docs/technical_reference/costing/stoichiometric_reactor.rst

Stoichiometric Reactor Costing Method
======================================

Currently, the costing method is implemented for lime and soda ash softening and acidification which only include

> the capital cost of building the reactor. The capital cost of lime soda ash is a function of 

Reviewer: The capital cost of lime **and** soda ash is a function...

total reagent mass being added to the softening process and is only valid when both precipitant and reagents are provided.

> While acid addition capital cost is only constructed if only reagents are provided. Acid addition costing is 

Reviewer: Acid addition capital cost **is constructed only if** reagents are provided.

Author: Done

> While acid addition capital cost is only constructed if only reagents are provided. Acid addition costing is 
> base on folume flow of acid per day. 

Reviewer 2: ```suggestion
While acid addition capital cost is only constructed if only reagents are provided. Acid addition costing is 
based on volume flow of acid per day. 
```

> (Please refer to to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_ for details on dissolution and precipitation reaction configurations). 

Reviewer 2: ```suggestion
(Please refer to the `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_ for details on dissolution and precipitation reaction configurations). 
```

.. csv-table::
   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"

   "Softening capital cost", ":math:`C_softening`", "``capital_cost_softening``", "374.9", ":math:`\text{USD/(lb/day)}`"

>    "Acid addition capital cost", ":math:`C_acid`", "``capital_cost_acid_addition``", "127.8", ":math:`\text{USD/(gallon/day)}`"

Reviewer 2: ```suggestion
   "Softening capital cost", ":math:`C_{softening}`", "``capital_cost_softening``", "374.9", ":math:`\text{USD/(lb/day)}`"
   "Acid addition capital cost", ":math:`C_{acid}`", "``capital_cost_acid_addition``", "127.8", ":math:`\text{USD/(gallon/day)}`"
```

Costing Method Variables
++++++++++++++++++++++++

> There are not unique costing variables constructed for stoichiometric reactor

Reviewer 2: ```suggestion
There are not unique costing variables constructed for the stoichiometric reactor.
```

Capital Cost Calculations
+++++++++++++++++++++++++

> (if user only includes reagents).(Refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_) 

Reviewer 2: ```suggestion
If user only includes reagents, refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_) 
```

    .. math::

        CC_{softening}=C_{softening}*\sum{M_{reagent}}

        CC_{acidification}=C_{acid}*\sum{Q_{reagent}}

>  where M_{reagent} is mass flow all reagent added in softening processes, and Q_{reagent} is flow volume of chemical added. 

Reviewer 2: ```suggestion
 where :math:`M_{reagent}` is mass flow all reagent added in softening processes, and :math:`Q_{reagent}` is flow volume of chemical added. 
```

Operating Cost Calculations
+++++++++++++++++++++++++++

> There are no operational costs provided, user needs to manually cost the reagent addition per mass/flow of reagent as in example below for 

Reviewer 2: ```suggestion
There are no operational costs provided, user needs to manually cost the reagent addition per mass/flow of reagent as done in the example below for 
```

> softening operation with soda ash and lime addition. 
> .. code-block::

Reviewer 2: ```suggestion
softening operation with soda ash and lime addition. 

.. code-block::
```

.. csv-table::
   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"

   "Softening capital cost", ":math:`C_{softening}`", "``capital_cost_softening``", "374.9", ":math:`\text{USD/(lb/day)}`"
   "Acid addition capital cost", ":math:`C_{acid}`", "``capital_cost_acid_addition``", "127.8", ":math:`\text{USD/(gallon/day)}`"

Costing Method Variables
++++++++++++++++++++++++

There are not unique costing variables constructed for the stoichiometric reactor.

Capital Cost Calculations
+++++++++++++++++++++++++

> If user only includes reagents, refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_) 

Reviewer 2: ```suggestion
If user only includes reagents, refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_
```
